table-api/src/main/java/io/deephaven/api/ColumnMatch.java

```diff
@@ -0,0 +1,33 @@
+package io.deephaven.api;
+
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Parameter;
+
+@Immutable(builder = false, copy = false)
+public abstract class ColumnMatch implements JoinMatch {
```

Comment:
Is `ColumnMatch` also a `Filter`?

Reply:
Yes - we should be able to have an AST where `ColumnMatch` is a `Filter`, but unfortunately we can't parse a generic string from Filter and get out a ColumnMatch, since the RHS depends on the query scope context.